src/NotificationMailing.php

```diff
@@ -82,18 +82,21 @@ public static function isUserAddressValid($address, $options = ['checkdns' => fa
     }
 
 
-    public static function testNotification()
+    /**
+     * Sends a test email to the administrator.
+     *
+     * @return array An array containing success, error and debug information
+     */
+    public static function testNotification(): array
     {
         global $CFG_GLPI;
 
         $sender = Config::getEmailSender();
         if ($sender['email'] === null || !self::isUserAddressValid($sender['email'])) {
-            Session::addMessageAfterRedirect(
-                __('Sender email is not a valid email address.'),
-                false,
-                ERROR
-            );
-            return false;
+            return [
+                'success' => false,
+                'error' => __('Sender email is not a valid email address.'),
```

Comment:
It is better to always have the same response "shema".
```suggestion
                'error' => __('Sender email is not a valid email address.'),
                'debug' => null,
```